lang/cs/Org.Apache.REEF.Tests/Functional/IMRU/IMRUCloseTaskTest.cs

```diff
@@ -60,22 +62,15 @@ public void TestTaskCloseOnLocalRuntime()
             const int numTasks = 4;
             var testFolder = DefaultRuntimeFolder + TestId;
             TestBroadCastAndReduce(false, numTasks, chunkSize, dims, iterations, mapperMemory, updateTaskMemory, testFolder);
-            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder);
-            var failedCount = GetMessageCount(lines, FailTaskMessage);
+            string[] lines = ReadLogFile(DriverStdout, "driver", testFolder, 120);
```

Comment:
Changed  memory  to 512. 
The test shows the close event usually is received in the first 1 or 2 iteration. So 200 should be sufficient. Here we are testing real IMRU tasks rather than test task, we don't want to add some signal logic in IMRU tasks to change the original code logic.